opentracing_instrumentation/local_span.py

```diff
@@ -22,7 +22,9 @@
 import functools
 import contextlib2
 import tornado.concurrent
-from . import get_current_span, span_in_stack_context, utils
+import opentracing
+from opentracing.scope_managers.tornado import TornadoScopeManager
```

Comment:
will this not blow up the lib for non-tornado users? (I don't know if there are such users, tbh)

Reply:
It doesn't cause a problem, because tornado will be installed with this library https://github.com/uber-common/opentracing-python-instrumentation/blob/master/setup.py#L37 Of course, it can be problem if we decide to move external libraries to extra_requires for example.